src/main/java/edu/harvard/iq/dataverse/DatasetPage.java

```diff
@@ -410,17 +410,13 @@ public DataFile getInitialDataFile(){
         return null;
     }
     
-    public String getSwiftContainerName(){
-
-        String swiftContainerName;
+    public SwiftAccessIO getSwiftObject() {
```

Comment:
I have a feeling it will get merged after, so we can deal with the merge conflicts & changes when that happens